api/v1beta1/machinehealthcheck_types.go

```diff
@@ -165,6 +166,7 @@ type MachineHealthCheckStatus struct {
 
 	// targets shows the current list of machines the machine health check is watching
 	// +optional
+	// +kubebuilder:validation:MaxItems=100000
```

Comment:
What is the maximum length of the string here? It's a machine name right, so 253 characters

If we assume the entire request is this list, then 3*1024*1024/100000, means the average length would be around 31.5 characters per entry in the list

I suspect this targets list may be able to be restricted slightly smaller.

Given this is the collection of machines, is 10,000 sufficient? (It would allow full 253 chars for each entry in the list in theory)

Reply:
I think 10k is fine. This would roughly be the limit for a single MD / MP then